### ISSUE TITLE
Use gem bundle for easier gem installation

### ISSUE THREAD

**Issue by aslakhellesoy** [cucumber-attic/cuke4duke#63]

**Comment by aslakhellesoy** [comment]
This is work in progress on [bracki's branch](http://github.com/aslakhellesoy/cuke4duke/network)

It still needs work - deleting ~/.m2/repository/.jruby and then following [build instructions](http://wiki.github.com/aslakhellesoy/cuke4duke/build-cuke4duke) fails. Furthermore, only the celerity maven example seems to be updated with the new gem element. All examples need to be updated, both for maven and ant.

**Comment by bracki** [comment]
Yep. Definitely. _-Dcucumber.installGems_ is broken in the current state. Work will resume next week.

**Comment by aslakhellesoy** [comment]
Any progress on this?

**Comment by bracki** [comment]
Not really. I found that packaging gems into jars works nicely. Unfortunately executing commands that are packaged into bin\somecommand.rb does not work. My favourited approach right now is to a pack a jar containing all needed gems and then unpack that to some/folder/.gemhome before running cuke4duke.

**Comment by nealeu** [comment]
This sounds similar to the approach I've taken to avoid -Dcucumber.installGems, which doesn't play well in a CI environment.
